docs/_adapters/cpx.md

```diff
@@ -8,33 +8,34 @@ project_status: beta
 github_link: https://github.com/layer5io/meshery-cpx
 image: /docs/assets/img/service-meshes/cpx.png
 ---
-# {{ page.name }}
 
-| Service Mesh   | Adapter Status | Latest Supported Mesh Version |
-| :------------: | :------------:   | :------------:              |
-| {{page.title}} | [{{ page.project_status }}]({{ page.github_link }}) | {{page.version}}  |
+# {{ page.name }}
 
+|  Service Mesh  |                   Adapter Status                    | Latest Supported Mesh Version |
+| :------------: | :-------------------------------------------------: | :---------------------------: |
+| {{page.title}} | [{{ page.project_status }}]({{ page.github_link }}) |       {{page.version}}        |
 
 ### Features
-1. Lifecycle Management of Citrix 
-2. Lifecycle Management of Sample Applications
-
 
-### Lifecycle management
-The {{page.name}} can install **{{page.version}}** of the {{page.name}} service mesh. The SMI adapter for Kuma can also be installed using Meshery.
+1. Lifecycle Management of Citrix
```

Comment:
Please update to "Lifecycle Management of Citrix Service Mesh", if you would.

Reply:
Actually, even better to update to "Lifecycle Management of {{ page.name }}"

Reply:
On it

Reply:
@leecalcote  I'm done. Could you review the changes?